crates/core/tests/operations.rs

```diff
@@ -69,7 +69,7 @@ async fn base_node_test_config(
             ignore_protocol_checking: true,
             address: Some(Ipv4Addr::LOCALHOST.into()),
             network_port: public_port,
-            bandwidth_limit: None,
+            bandwidth_limit: Some(12_500_000), // 10 Mbps = 1.25 MB/s * 10s window
```

Comment:
this is a very arbitrary number, each setup may differ from what bandwidth it  has available.

also last time I checked using this actually slowed down transfer significantly (2 orders of magnitude), I don't think it was quite working as expected